cibyl/utils/arguments.py

```diff
@@ -0,0 +1,41 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+from typing import Dict, List
+
+from cibyl.cli.argument import Argument
+from cibyl.utils.dicts import subset
+
+
+def has_filtering_arguments(args: Dict[str, Argument],
+                            args_to_check: List[str]) -> bool:
+    """Check if the args dictionary contains any argument with values to
+    filter, specified in of the args_to_check list.
+
+    :returns: Whether there is any argument in args, of the ones specified in
+    the args_to_check list, that has a value to filter the output.
+    """
+    test_args = subset(args, args_to_check)
+    return any(arg.value for arg in test_args.values())
```

Comment:
Have you thought about valueless filters like '--last-build'. I do not know if that interferes here.

Reply:
Right now, `--last-build` should not affect the use case, since it will only affect build that have at least one build, and this function will be used to remove jobs that have no builds at all and there is no valueless argument for tests, but that is a good point.